dotcom-rendering/src/web/components/FixedMediumSlowVI.tsx

```diff
@@ -0,0 +1,124 @@
+import { ArticleDesign } from '@guardian/libs';
+import { Card } from './Card/Card';
+import { LI } from './Card/components/LI';
+import { UL } from './Card/components/UL';
+
+type Props = {
+	trails: TrailType[];
+	containerPalette?: DCRContainerPalette;
+	showAge?: boolean;
+};
+
+export const FixedMediumSlowVI = ({
+	trails,
+	containerPalette,
+	showAge,
+}: Props) => {
+	const topTrails = trails.slice(0, 2);
+	const bottomTrails = trails.slice(2, 6);
+
+	return (
+		<>
+			<UL direction="row" padBottom={true}>
+				{topTrails.map((trail, index) => {
+					return (
+						<LI
+							key={trail.url}
+							padSides={true}
+							showDivider={index > 0}
+							padBottomOnMobile={index === 0 ? true : false}
```

Comment:
I can see your argument, and I think that the considerations you mentioned are definitely relevant. However, my feeling is that there is an advantage to this pattern, even with just two cards. The two of them do share a lot of props in common, and having the logic inlined here makes it easier to see which props are different, and hence responsible for the differences in their behaviour. I think if there were more than a handful of props being set in this way, or if there were more complex rules deciding the designs of the cards, then inlining would quickly get unmanageable, but my feeling is that it's okay in this case.

So the argument in my mind for using a map here isn't for DRY purposes, but rather that it (hopefully) makes it easier to see what the two cards have in common than it would be if they were hard-coded separately. If you or others have strong feelings about this case then I'm happy to change it though, lmk what you think?

Reply:
Sounds fair 👍  You make a good case and I don't have strong feelings so let's go map!